src/quo2/components/navigation/bottom_nav_tab.cljs

```diff
@@ -43,52 +44,54 @@
+               :holes (cond
+                        (not new-notifications?) ;; No new notifications, remove holes
+                        []
 
-                      (= notification-indicator :unread-dot)
-                      [{:x 50 :y 5 :width 10 :height 10 :borderRadius 5}]
+                        (= notification-indicator :unread-dot)
+                        [{:x 50 :y 5 :width 10 :height 10 :borderRadius 5}]
 
-                      :else
-                      [{:x 47 :y 1 :width 18 :height 18 :borderRadius 7}])}
-            [reanimated/image
-             {:style  icon-animated-style
-              :source (icons/icon-source (keyword (str icon 24)))}]]
-           [reanimated/image
-            {:style  icon-animated-style
-             :source (icons/icon-source (keyword (str icon 24)))}])
-         (when new-notifications?
-           (if (= notification-indicator :counter)
-             [counter/counter
-              {:override-text-color colors/white
-               :override-bg-color   colors/primary-50
-               :style               {:position :absolute
-                                     :left     48
-                                     :top      2}}
-              counter-label]
-             [rn/view
-              {:style {:width            8
-                       :height           8
-                       :border-radius    4
-                       :top              6
-                       :left             51
-                       :position         :absolute
-                       :background-color colors/primary-50}}]))]]))])
+                        :else
+                        [{:x 47 :y 1 :width 18 :height 18 :borderRadius 7}])}
+              [reanimated/image
+               {:style  icon-animated-style
+                :source (icons/icon-source (keyword (str icon 24)))}]]
+             [reanimated/image
+              {:style  icon-animated-style
+               :source (icons/icon-source (keyword (str icon 24)))}])
+           (when new-notifications?
+             (if (= notification-indicator :counter)
+               [counter/counter
+                {:override-text-color colors/white
+                 :override-bg-color   colors/primary-50
+                 :style               {:position :absolute
+                                       :left     48
+                                       :top      2}}
+                counter-label]
+               [rn/view
+                {:style {:width            8
+                         :height           8
+                         :border-radius    4
+                         :top              6
+                         :left             51
+                         :position         :absolute
+                         :background-color colors/primary-50}}]))]]
+         double-tap-gesture (conj [gesture/gesture-detector {:gesture double-tap-gesture}]))))])
```

Comment:
This `conj` is unnecessary.

Reply:
It's gonna be added as the last element of the vector from the thread-last macro automatically.
Amazing point!
So no more `conj`ing to conditionally add to vectors :)

Reply:
Unfortunately, This didn't work.
I had to use conj again.
I was getting this
![image](https://user-images.githubusercontent.com/33176106/230911587-aff9210f-36ae-4bdc-9d7e-ef9aad27a666.png)